File: Shoping-list/Shoping-List.py
##Shoping a List 
import os
#new open to apps
def welcome():
    print_Welcome = 'Welcome to apps' 
    print(print_Welcome.center(80))
welcome()
#help of apps
def help():
    
    print('''-->Hi Iam one robat the apps<--
                **my name is 'Atom'**
            to apps you have a one List and more.
            you can a Enter 'shwo' to show your List,
            sure!, you can Enter 'count' to number a items your List
            for clearing a display enter clear
            and you can Enter 'exit' to exit a apps
                 **a nice your times**
''')
help()    
#the List of a empty 
Shoping_List = []

#shoping function
def Shoping():
    while True:
        new = input('--> ')
        Shoping_List.append(new)
        if new == 'exit':
            break
        elif new == "help":
            help()
            Shoping
        elif new == "clear":
            os.system("clear")
        elif new == 'show':
            Shoping_List.pop()
            for item in Shoping_List:
                print(f"[+]- {item}")
        elif new == 'count':
            Shoping_List.pop()
            print('it is number of your list ---->',len(Shoping_List))   
Shoping()

File: Shoping-list/Shoping-List.rb
def help
  help="""\t\t ==> welecome to shoping list app <== !!
you can enter all thing to save on your shop list.
sure!, use the show to show item of your list
and you can use the count to show the  count of item on your list
and use the clear to cleared display app
for exit of app , use exit\n
\t\t ***THANKS YOU FOR SEE MY !!*** \n
"""
 puts help
end
def main
  system("clear")
  text="""\t\t ==> welcome to Shoping List <==\n
Hi, my name is 'atom',i'am first robot on app !
this is app for lists the all things for Shoping
for see the more of help you can use the help\n
"""
  shop=[]
  delete=[]
  puts text
  while true
      print"SAJAD ~$ "
      command=gets.chomp
      if command == "exit"
        break
      elsif command == "clear"
        system("clear")
      elsif command == "show"
        for i in shop
          puts "[+] #{i}"
        end
      elsif command == "help"
        help
      elsif command == "count"
         count=0
         for i in shop
           count+=1
         end
         puts"the count of item on shop lists ==> #{count}"

      else
         shop.append(command)
      end
  end
end
main
